cmd/janitor/gcp_janitor.py

```diff
@@ -72,6 +72,82 @@
     Resource('', 'logging', 'sinks', None, None, None, False, False),
 ]
 
+# gcloud compute zones list --format="value(name)" | sort | awk '{print "    \x27"$1"\x27," }'
+ZONES = [
```

Comment:
By the way, what is the reason for this to be defined liked that, and not be resolved in runtime via API call?

Reply:
Ah, just saw that comment about outages, disregard that.

Reply:
We've had issues in the past where the list of all zones computed by gcloud itself (likely using this same API call, though I'm not sure) doesn't match the list of zones known to things like the compute endpoint, which causes it to fail.